### PR TITLE
chore(upload-assets-to-s3): upload svg files

### PR DESCRIPTION
**Pull request by antonsamper**
#### Reference
- https://financialtimes.atlassian.net/browse/CPP-230

#### Notes
- Add `svg` to list of extensions to upload

### CODE REVIEW

**Review by nickcolley — APPROVED** [review]
Makes sense 👍

File: tasks/upload-assets-to-s3.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const aws = require('aws-sdk');
 const defaultDirectory = 'public';
 const defaultBucket = 'ft-next-hashed-assets-prod';
 const defaultDestination = 'hashed-assets/uploaded';
-const defaultFileExtensions = ['js', 'css', 'map', 'gz', 'br', 'png', 'jpg', 'jpeg', 'gif', 'webp'].join();
```
Comment (nickcolley) [inline tasks/upload-assets-to-s3.js:10]: since these are the defaults can also consider extending them if it's only required in one place.

Comment (apaleslimghost) [reply tasks/upload-assets-to-s3.js:10]: this command is called by `n-gage` which doesn't currently offer a way to set this argument, so the app would have to override that task in its makefile to add the argument, which makes me queasy

i think SVG would be a common enough use case that it should be a default?

Comment (antonsamper) [reply tasks/upload-assets-to-s3.js:10]: Yeah that makes sense but it would be a bit more involved, something to consider next time

Comment (nickcolley) [reply tasks/upload-assets-to-s3.js:10]: > this command is called by n-gage which doesn't currently offer a way to set this argument

Didn't realize that, makes sense to me.

**Pull request merged**